map/map_loader/test/lanelet2_map_loader_launch.test.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 TIER IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from ament_index_python import get_package_share_directory
+import launch
+from launch import LaunchDescription
+from launch_ros.actions import Node
+import launch_testing
+import pytest
+
+
+@pytest.mark.launch_test
+def generate_test_description():
+
+    lanelet2_map_path = os.path.join(
+        get_package_share_directory("map_loader"), "test/data/test_map.osm"
+    )
+
+    lanelet2_map_loader = Node(
+        package="map_loader",
+        executable="lanelet2_map_loader",
+        namespace="had_maps",
```

Comment:
[nits] I think `namespace` isn't necessary here. If you write `had_maps`, then people might wonder "what is `had_maps`?:thinking:".

```suggestion
        namespace="",
```

Reply:
I removed it: [1832c77](https://github.com/autowarefoundation/autoware.universe/pull/1056/commits/1832c7758edcb3dbce23a576019bc47304ac5c3e)